Update/some_004.txt

```diff
@@ -3532,14 +3532,14 @@ void main()
 			NULL, " ...In here, earlier,", Line_WaitForInput);
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300304", 256, TRUE);
 	OutputLine(NULL, "…暁、いたでしょ…？！」",
-			NULL, " ...it was Akira, right...?!\"", Line_Normal);
+			NULL, " ...It was Akira, right...?!\"", Line_Normal);
 	ClearMessage();
 
 //夏美rvS04/36/310100422.「っ————」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100422", 256, TRUE);
 	OutputLine(NULL, "「っ————」",
-			NULL, "\"...\"", Line_Normal);
+			NULL, "\"uh————\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"Uh...\"", Line_Normal);
```
See note on line #2634. And since it's the start of a new thought — or the lack of one :) — "uh" should be "Uh"